signer/contentsignaturepki/x509.go

```diff
@@ -146,3 +146,11 @@ func (s *ContentSigner) makeChain() (chain string, name string, err error) {
 	name = fmt.Sprintf("%s-%s.chain", cert.Subject.CommonName, cert.NotAfter.Format("2006-01-02-15-04-05"))
 	return
 }
+
+func (s *ContentSigner) domainForLeafCert() string {
+	subdomain := s.ID
+	if s.subdomainOverride != "" {
```

Comment:
What's the value if the config does not contain a `SubdomainOverride` value? Is it `""` or `nil`?

Reply:
It's a `""`. That's the Go zero value for strings.